src/components/Match/MatchLog.jsx

```diff
@@ -118,6 +119,69 @@ const logColumns = [heroTdColumn, {
           />
         );
       }
+      case 'objectives': {
+        if (row.detail === ' First Blood') {
+          return (
+            <span>
+              <img
+                src="/assets/images/dota2/bloodsplattersmall.png"
+                alt=""
+                style={{ height: '30px', float: 'left', paddingTop: '7px' }}
+              />
+              <p>{strings.th_firstblood_claimed}</p>
+            </span>
+          );
+        }
+        if (tb.radiant[row.key]) {
+          return (
+            <span>
+              <img
+                src="/assets/images/dota2/dotastructuresmallradiant.png"
```

Comment:
The structure icon looks kind of strange--maybe just use the IconRadiant and IconDire that we already have?